apps/condo/domains/acquiring/schema/MultiPayment.test.js

```diff
@@ -212,24 +252,424 @@ describe('MultiPayment', () => {
+                    await updateTestAcquiringIntegration(admin, acquiringIntegration.id, {
+                        canGroupReceipts: true,
+                    })
+                    const [multiPayment] = await createTestMultiPayment(admin, payments, client.user, acquiringIntegration)
+                    await updateTestPayment(admin, payments[0].id, {
+                        status: PAYMENT_DONE_STATUS,
+                    })
+                    await updateTestPayment(admin, payments[1].id, {
+                        status: PAYMENT_DONE_STATUS,
+                    })
+                    await expectToThrowValidationFailureError(async () => {
+                        await updateTestMultiPayment(admin, multiPayment.id, {
+                            ...multiPaymentDonePayload,
+                            implicitFee,
+                        }, MULTIPAYMENT_IMPLICIT_FEE_MISMATCH)
+                    })
+                })
+                test('All payments should not be deleted', async () => {
+                    const { admin, payments, client, acquiringIntegration } = await makePayerAndPayments(1)
+                    const [multiPayment] = await createTestMultiPayment(admin, payments, client.user, acquiringIntegration)
+                    await updateTestPayment(admin, payments[0].id, {
+                        status: PAYMENT_DONE_STATUS,
+                        deletedAt: dayjs().toISOString(),
+                    })
+                    await expectToThrowValidationFailureError(async () => {
+                        await updateTestMultiPayment(admin, multiPayment.id, multiPaymentDonePayload)
+                    }, MULTIPAYMENT_DELETED_PAYMENTS)
                 })
             })
         })
     })
     describe('real-life cases', () => {
-        // TODO(DOMA-1452) write tests
+        describe('UPS', () => {
+            let integrationClient
+            let registerMPResult
+            beforeEach(async () => {
+                const {
+                    admin,
+                    acquiringIntegration,
+                    billingReceipts,
+                    serviceConsumer,
+                    client,
+                } = await makePayer(1)
+                integrationClient = await makeClientWithNewRegisteredAndLoggedInUser()
+                await createTestAcquiringIntegrationAccessRight(admin, acquiringIntegration, integrationClient.user)
+                const [obj] = await registerMultiPaymentByTestClient(client, [
+                    { consumerId: serviceConsumer.id, receiptsIds: [billingReceipts[0].id] },
+                ], { sender: client.userAttrs.sender })
+                registerMPResult = obj
+            })
+            test('Successful payment', async () => {
+                // Stage 1. User goes to service and confirm payment with fees...
+                // After that Acquiring Integration Manager move MP and P to PROCESSING and setting explicit fees
```

Comment:
What is Acquiring Integration Manager?

Reply:
Is it our UPS-like service? That way i'd rather made the descriptions more generic:

`After polling bank services our acquiring integration service does ... ` 

But its your call

Reply:
changed `Acquiring Integration Manager` to `our acquiring integration service` inside description